CASMINST-4663 (#1824)
Cray-HPE/docs-csm: upgrade/1.2/Stage_0_Prerequisites.md

1. (`ncn-m001#`) Set the `CSM_RELEASE` variable to the **target** CSM version of this upgrade.

   ```bash
   CSM_RELEASE=1.2.0
   ```

1. If there are space concerns on the node, then add an `rbd` device on the node for the CSM tarball.

    This same `rbd` device can be remapped to `ncn-m002` later in the upgrade procedure, when the CSM tarball is needed on that node.

    **Note:** The `prepare-assets.sh` script will delete the CSM tarball. Copy it to a different location before running that script.
    Otherwise it will need to be downloaded again later in the upgrade procedure.

>     See [Alternate Storage Pools](../../operations/utility_storage/Alternate_Storage_Pools.md).

Reviewer: ```suggestion
    See [Create a storage pool](../../operations/utility_storage/Alternate_Storage_Pools.md#create-a-storage-pool)
    and [Create and map an `rbd` device](../../operations/utility_storage/Alternate_Storage_Pools.md#create-and-map-an-rbd-device).
```

    CSM_RELEASE=1.2.0
   ```

1. If there are space concerns on the node, then add an `rbd` device on the node for the CSM tarball.

    This same `rbd` device can be remapped to `ncn-m002` later in the upgrade procedure, when the CSM tarball is needed on that node.

> 

Reviewer: ```suggestion

    **Note:** The `prepare-assets.sh` script will delete the CSM tarball. Copy it to a different location before running that script.
    Otherwise it will need to be downloaded again later in the upgrade procedure.

```

    CSM_RELEASE=1.2.0
   ```

> 1. If there are space concerns on the node please see the process for adding an rbd device to store the CSM tarball [here](../operations/utility_storage/Alternate_Storage_Pools.md).

Reviewer: ```suggestion
1. If there are space concerns on the node, add an `rbd` device on the node for the CSM tarball.

    This same `rbd` device can be remapped to `ncn-m002` later in the upgrade procedure, when the CSM tarball is needed on that node.

    See [Alternate Storage Pools](../../operations/utility_storage/Alternate_Storage_Pools.md).
```
